SketchUp/Sketchup/ComponentInstance.rb

```diff
@@ -477,8 +477,8 @@ def split(instance)
   def subtract(instance)
   end
 
-  # The transform! method is used to apply a transformation to a component
-  # instance.
+  # The +transform!+ method is used to apply a {Geom::Transformation} to a
```

Comment:
I've started denoting all references to classes and methods with the reference tag. In this case I would write:

```
The {#transform!} method ...
```